gnovm/cmd/gno/test.go

```diff
@@ -295,7 +298,10 @@ func gnoTestPkg(
 				m.Alloc = gno.NewAllocator(maxAllocTx)
 			}
 			m.RunMemPackage(memPkg, true)
+			bGasUsed := m.GasMeter.GasConsumed()
```

Comment:
Can you be explicit with the `bGasUsed` and `aGasUsed` names as you were above? `beforeGasUsed` and `afterGasUsed`